src/algorithms/banditpam.cpp

```diff
@@ -104,6 +106,7 @@ namespace km {
       }
       updated_sigma(i) = arma::stddev(sample);
     }
+    batchSize = 100;
```

Comment:
Can you add a comment explaining that you're resetting it to it's original value?

Also I think you should save the original value in a variable and reset it to that here, instead of just using `100` (which might change later)

Reply:
Added 

```
// reset batchSize the original batch size as it's a global variable
// used by other functions (e.g. buildTarget, swapTarget)
batchSize = originalBatchSize;
```